src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (a, b) {
+      return a + b;
+    },
+
+    subtract: function (a, b) {
+      return a - b;
+    },
+
+    multiply: function (a, b) {
+      return a * b;
```

Comment:
[CODE KNOWLEDGE] - The `multiply` method should also use the shorthand syntax. Change `multiply: function (a, b) { return a * b; }` to `multiply(a, b) { return a * b; }`.